common/bin/oo-diagnostics

```diff
@@ -1204,6 +1204,33 @@ class OODiag
 
   end
 
+  def test_auth_keys_salt_failure
+    log_dir = "/var/log/openshift/broker/"
+
+    if Rails.env.development?
+      mode = "development"
+      msg_log = `grep 'Broker key authentication failed' #{log_dir + mode}.log`
+    else
+      mode = "production"
+      msg_log = `grep 'Broker key authentication failed' #{log_dir + mode}.log`
+    end
```

Comment:
There are three rails environments used: "development", "test", and "production". If rails is using the "test" environment, your current code will still look in production.log even though it may not exist or have the relevant log messages. Perhaps a case statement is needed.